Cython/Compiler/Visitor.py

```diff
@@ -675,7 +675,8 @@ def _dispatch_to_method_handler(self, attr_name, self_arg,
         method_handler = self._find_handler(
             "method_%s_%s" % (type_name, attr_name), kwargs)
         if method_handler is None:
-            if (attr_name in TypeSlots.method_name_to_slot
+            # in this case we don't need the real directives on the call to get_slot_table
```

Comment:
This comment doesn't feel helpful at this place. Using `special_method_names` is perfectly explicit and readable. And if we ever come up with a directive that changes the list of special names, then there's probably more to change than the code here.
```suggestion
```

Reply:
Yeah the comment was left over from the previous version so it can go.